dpnp/dpnp_iface_bitwise.py

```diff
@@ -59,11 +59,11 @@
 ]
 
 
-def use_dpnp_backend(x1, x2, valid_types):
+def use_dpnp_backend(x1, x2):
```

Comment:
It looks like this function is similar to `use_origin_backend`
perhaps it will be better to use `use_origin_backend` instead duplication